css/css-sizing/aspect-ratio/block-aspect-ratio-036.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: div block size aspect ratio precision</title>
+<link rel="author" title="Google LLC" href="https://www.google.com/">
+<link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht" />
```

Comment:
Could you add a "<meta assert" explaining what the test is checking exactly? Thanks.

Reply:
Done.